mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
These are simply created by the command `mvn wrapper:wrapper -Dmaven=3.8.6` to bootstrap maven wrapper initially.
Later installed maven is not needed.

I consider these files like a binary which is not for editing or reading.

One does not need to install maven at all, no M2_HOME, no adding mvn to path, no apt get install mvn, no maven version problems, because the version is defined by the source code in .mvn/maven-wrapper.properties file.

The idea is simpler redistribution. Overall this just lowers the entry bar to build the source a bit.

Still, one _can_ build the source with regular maven. I can add the instructions of how to do so. (Basically it is just `./mvnw` replaced with regular `mvn` binary no more than that).

Reply:
i like these wrappers, i like how they worked instantly and i liked how they are generated from a maven target.